shared/redux/rtk-apis/tasksAPI.types.ts

```diff
@@ -0,0 +1,18 @@
+import { EPriority, ETaskStatus } from "@/shared/typedefs/enums";
+
+export type TCreateTaskPayload = {
+  title: string;
+  description?: string;
+  dueDate?: Date | null;
+  priority: EPriority;
+  status: ETaskStatus;
+};
```

Comment:
Should not be seeing changes from other PRs. Please be mindful of rebasing in the future